dataprofiler/labelers/char_load_tf_model.py

```diff
@@ -18,14 +22,18 @@
 logger = dp_logging.get_child_logger(__name__)
 labeler_utils.hide_tf_logger_warnings()
 
+Data = Union[pd.DataFrame, pd.Series, np.ndarray]
```

Comment:
@tonywu315 love it --can we make this used everywhere? I see other places in other files where `Union[pd.DataFrame, pd.Series, np.ndarray]` is used but not with an alias. I'd recommend doing this so we don't have to repeat in other files too

Reply:
added to `utils.py`